helmfile/values/kube-prometheus-stack-wc.yaml.gotmpl

```diff
@@ -32,6 +31,15 @@ kube-state-metrics:
 
 prometheus-node-exporter:
   resources: {{- toYaml .Values.prometheusNodeExporter.resources | nindent 4 }}
+  rbac:
+    pspEnabled: true
```

Comment:
Should we enable psp? since it will be removed soon:tm:

Reply:
Yes enable it and let the task of migrating to psa disable it.